package.json

```diff
@@ -16,7 +16,9 @@
     "dev": "yarn build -- --watch src",
     "build": "babel src --out-dir dst --ignore '**/*.test.js'",
     "lint": "eslint \"{src,example}\"",
-    "test": "yarn lint && jest",
+    "test": "yarn lint && yarn test:unit${TRAVIS:+':coveralls'}",
+    "test:unit": "jest --coverage",
+    "test:unit:coveralls": "jest --coverage --coverageReporters=text-lcov | coveralls",
```

Comment:
Just curious. I can't find `coverageReporters` option in the jest docs - is this something they support unofficially?

Reply:
Good question - this is taken from the [node-coveralls docs](https://github.com/nickmerwin/node-coveralls#jest).  I'll see if I can dig up some docs.

Reply:
From the top paragraph of the [Jest CLI docs](https://jestjs.io/docs/en/cli):

> Every one of Jest's Configuration options can also be specified through the CLI.
 
So it looks like the docs for the CLI only cover CLI-specific things, but everything else from config (including [coverageReporters](https://jestjs.io/docs/en/configuration#coveragereporters-array-string)) is also supported via CLI.